overlord/fdestate/fdestate.go

```diff
@@ -157,39 +156,11 @@ type KeyslotRoleInfo struct {
 	TPM2PCRPolicyRevocationCounter uint32 `json:"tpm2-pcr-policy-revocation-counter,omitempty"`
 }
 
-type hashAlg crypto.Hash
```

Comment:
what if we move this code into the nonsecboot build? that way we keep the tests working with little change, but we the actual type is moved back to snapd/secboot package. I think this should be preceded then by a comment that we intend to keep it compatible with the way snapcore/secboot serialization works.